File: benchmarks/go/sort.go
package main

import (
	"fmt"
	"sort"
)

func main() {
	values := []int64{}
	seed := int64(12345)
	for i := 0; i < 300000; i++ {
		seed = seed * 48271 % 2147483647
		values = append(values, seed%1000000)
	}
	sort.Slice(values, func(a, b int) bool { return values[a] < values[b] })
	fmt.Println(values[0], values[len(values)/2], values[len(values)-1])
}

File: benchmarks/java/Sort.java
import java.util.*;
public class Sort {
    public static void main(String[] args) {
        long[] values = new long[300000];
        long seed = 12345;
        for (int i = 0; i < values.length; i++) {
            seed = seed * 48271 % 2147483647;
            values[i] = seed % 1000000;
        }
        Arrays.sort(values);
        System.out.println(values[0] + " " + values[values.length / 2] + " " + values[values.length - 1]);
    }
}
